test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -1131,8 +1146,11 @@ void UcxConnection::connect_tag(UcxCallback *callback)
     }
 
     // send local connection id
-    void *sreq = ucp_stream_send_nb(_ep, &_conn_id, 1, dt_int,
-                                    common_request_callback, 0);
+    param.cb.send = common_request_callback_nbx;
```

Comment:
can we remove the existing common_request_callback, and rename common_request_callback_nbx to common_request_callback?